### PR TITLE
Remove incorrect README.md instruction

### PR DESCRIPTION
**Pull request by oscr**
This step doesn't work (and isn't needed) since you're already in the cocos2d-x folder.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,7 +36,6 @@ Git user attention
 
 3. After running `download-deps.py`.
 
-         $ cd cocos2d-x
```
Comment (pandamicro) [inline README.md:39]: It's more like a hint to let user know where the next command should be executed

Comment (zilongshanren) [inline README.md:39]: @oscr  I agree with @pandamicro

Comment (oscr) [inline README.md:39]: @pandamicro @andyque Thank you for your quick feedback!

If you just copy-paste the instructions from the readme you will get an error:

```
oscr@oscr-VirtualBox:~/temp/cocos2d-x$ python download-deps.py
(...)
oscr@oscr-VirtualBox:~/temp/cocos2d-x$ cd cocos2d-x
bash: cd: cocos2d-x: No such file or directory
```

This actually caused me some confusion when I got started. If we want the user to know where (s)he should be when a command is executed then maybe we could hint this by letting the command prompt show the current catalog? For example:

```
$ git clone https://github.com/cocos2d/cocos2d-x.git
$ cd cocos2d-x
cocos2d-x $ python download-deps.py
cocos2d-x $ git submodule update --init
```

Comment (pandamicro) [inline README.md:39]: Oh, I see, there is another `cd cocos2d-x` in the previous instruction.

I think it's ok to delete this line, but can you append `cocos2d-x` in the front of `$`, that would be helpful, thank you

**Pull request merged**